package.json

```diff
@@ -17,9 +17,10 @@
     "date-utils": "~1.2.12"
   },
   "devDependencies": {
-    "should": "~1.2.1",
+    "bluebird": "^2.10.1",
```

Comment:
In fact, it´s supposed to be a dependency instead of a devDependency and the reason is that i use the "dual api" support from this lib (nodeify), to add support to promises to this lib. I´ll move this dependency to the correct place. A new release number hasn´t been generated yet, so we can still fix it.